src/main/kotlin/com/hyperdevs/poeditor/gradle/network/PoEditorApiController.kt

```diff
@@ -56,17 +58,20 @@ class PoEditorApiControllerImpl(private val apiToken: String,
         return response.onSuccessful { it.result.languages }
     }
 
+    @Suppress("LongParameterList")
     override fun getTranslationFileUrl(projectId: Int,
                                        code: String,
                                        type: ExportType,
                                        filters: List<String>?,
+                                       order: String?,
                                        tags: List<String>?): String {
         val response = poEditorApi.getExportFileInfo(
             apiToken = apiToken,
             id = projectId,
             type = type.toString().toLowerCase(),
             filters = filters,
             language = code,
+            order = order,
```

Comment:
If you use the enum approach, I would do something like this so we can pass the `NONE` value as null to the API:
```
            order = order.takeIf { it != OrderType.NONE }?.toLowerCase(), 
```